pkg/ingester-rf1/stream.go

```diff
@@ -246,14 +247,18 @@ func (s *stream) validateEntries(ctx context.Context, entries []logproto.Entry,
 		}
 
 		lineBytes := len(entries[i].Line)
+		metadataBytes := 0
+		for _, metadata := range entries[i].StructuredMetadata {
```

Comment:
can we use `util.StructuredMetadataSize(entry.StructuredMetadata)` here?

Reply:
oh, forgot to replace it. thanks

Reply:
done